pkgs/misc/dart-sass-embedded/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, system
+, stdenvNoCC
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "dart-sass-embedded";
+  version = "1.62.0";
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  nativeBuildInputs = [ ] ++ lib.optional stdenvNoCC.hostPlatform.isLinux autoPatchelfHook;
+
+
+  src = fetchurl {
+    url = {
+      "x86_64-linux" = "https://github.com/sass/dart-sass-embedded/releases/download/${version}/sass_embedded-${version}-linux-x64.tar.gz";
+      "aarch64-linux" = "https://github.com/sass/dart-sass-embedded/releases/download/${version}/sass_embedded-${version}-linux-arm64.tar.gz";
+      "x86_64-darwin" = "https://github.com/sass/dart-sass-embedded/releases/download/${version}/sass_embedded-${version}-macos-x64.tar.gz";
+      "aarch64-darwin" = "https://github.com/sass/dart-sass-embedded/releases/download/${version}/sass_embedded-${version}-macos-arm64.tar.gz";
+    }."${system}";
+    hash = {
+      "x86_64-linux" = "sha256-szSKhLnUan1dNjwkDbcZHVyML81MtGTQ2d9KA41eQo4=";
+      "aarch64-linux" = "sha256-W8kmTvZhLEQes0LCA/udzsYpbGIAe68qqNDtTdovIRA=";
+      "x86_64-darwin" = "sha256-LS2kd8mkEtdFMkeT3qEDpd08Ie2Dy+7uaY/0/wuHGLU=";
+      "aarch64-darwin" = "sha256-LEd7QZDPFkqBzBUxXEpBFyCpHYtoHIvHsKwmK1XbHro=";
+    }."${system}";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r * $out
+    ln -s $out/dart-sass-embedded $out/bin/dart-sass-embedded
+  '';
+
+  meta = {
+    description = "A wrapper for Dart Sass that implements the compiler side of the Embedded Sass protocol";
+    homepage = "https://github.com/sass/dart-sass-embedded";
+    changelog = "https://github.com/sass/dart-sass-embedded/blob/${version}/CHANGELOG.md";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ shyim ];
+  };
```

is more common I think

```suggestion
  meta = with lib; {
    description = "A wrapper for Dart Sass that implements the compiler side of the Embedded Sass protocol";
    homepage = "https://github.com/sass/dart-sass-embedded";
    changelog = "https://github.com/sass/dart-sass-embedded/blob/${version}/CHANGELOG.md";
    license = licenses.mit;
    maintainers = with maintainers; [ shyim ];
  };
```